_data/projects/CASE.yml

```diff
@@ -0,0 +1,14 @@
+name: CASE
+desc: A frameworkless and Open Source tool to build CRUD applications as dashboards or admin panel.
+site: https://case.app
+tags:
+  - appbuilder
+  - typescript
+  - adminpanel
+  - crud
+  - frameworkless
+  - opensource
+  - lowcode
+upforgrabs:
+  name: good first issue
+  link: https://github.com/casejs/CASE/labels/good%20first%20issue
```

Comment:
At the moment there are no issues assigned to this label, which means it won't be listed in the search results.

Is this something you're able to update (either now or in the future)?

Reply:
@shiftkey Yes, we will add issues assigned to this label tomorrow.

Thank you